crowbar_framework/app/assets/javascripts/barclamps/neutron/application.js

```diff
@@ -227,73 +228,44 @@
   };
 }(jQuery, document, window));
 
+
 $(document).ready(function($) {
-  $('#networking_plugin').on('change', function() {
-    var value = $(this).val();
-    var networking_mode = $('#networking_mode')
-    var non_forced_mode = networking_mode.data('non-forced');
-
-    switch (value) {
-      case 'linuxbridge':
-        if (non_forced_mode == undefined) {
-          networking_mode.data('non-forced', networking_mode.val());
-        }
-        networking_mode.val('vlan').trigger('change');
-        $('#mode_container').hide(100).attr('disabled', 'disabled');
-
-        $('#vmware_container').hide(100).attr('disabled', 'disabled');
-        break;
-      case 'openvswitch':
-        if (non_forced_mode != undefined) {
-          networking_mode.val(non_forced_mode);
-          networking_mode.removeData('non-forced');
+    $('#networking_plugin').on('change', function() {
+        var plugin = $(this).val();
```

Comment:
> And this var definition is useless as the variable gets used only once.

Done in https://github.com/toabctl/barclamp-neutron/commit/bb6f0757b4155c9fb84e87254977e84028a66cbc